Formula/a/alpscore.rb

```diff
@@ -18,24 +18,27 @@ class Alpscore < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "a6159afbf12d22d9d08a30e69a785d9bb77e595145937857e5e8c3ceb273582d"
   end
 
-  depends_on "cmake" => :build
+  depends_on "cmake" => [:build, :test]
   depends_on "boost"
   depends_on "eigen"
   depends_on "hdf5"
   depends_on "open-mpi"
 
-  on_macos do
-    depends_on "libaec"
-  end
-
   def install
-    args = std_cmake_args
-    args << "-DEIGEN3_INCLUDE_DIR=#{Formula["eigen"].opt_include}/eigen3"
-    args << "-DALPS_BUILD_SHARED=ON"
-    args << "-DENABLE_MPI=ON"
-    args << "-DTesting=OFF"
+    # Work around different behavior in CMake-built HDF5
+    inreplace "common/cmake/ALPSCommonModuleDefinitions.cmake" do |s|
+      s.sub! "set(HDF5_NO_FIND_PACKAGE_CONFIG_FILE TRUE)", ""
+      s.sub! "find_package (HDF5 1.10.2 ", "find_package (HDF5 "
+    end
```

Comment:
Should we report this upstream?

Reply:
Done here: https://github.com/ALPSCore/ALPSCore/issues/564#issuecomment-2329931903

We worked around the workaround :)